### ISSUE TITLE
Proposal peer review

### ISSUE THREAD

**Issue by win13676** [Yazmine1213C/ORIE5741_Project#2]
The team is proposing to use Graph Neural Network to classify music genre because audio data is non-Euclidean and GNN could perform better on non-Euclidean domains. The team also plan to use Siamese Networks to extract feature on the image of the audio spectrogram and attempt to modify the network to improve the accuracy or use less features. Then the team will use GNN to model the features and experiment with different aggregator functions. Finally the team will use a small network to compute similarity score. The team plan to use GTZAN dataset which consists multiple audio files from different genre.

Exploring different network and tackling audio dataset is very interesting. The audio/spectogram image data are continuous depend on it's neighbors so some data might be lost when mapped into linear space to compute Euclidean distance. The team provided a good background and justification why GNN might be a good choice to tackle audio classification. I am convinced that GNN would be a good network to explore audio data. The dataset is also good. It contains 10 genre with 30 seconds 100 audio files. There're many genre to classify and contains sufficient training data. It is also interesting that the team will be using Siamese Networks to extract features and also explore improvements on it. 

The report seems to be incomplete, especially in the Proposed Methods section.  There are several incomplete sentences and the explanation isn't very clear. The report also went above the 1 page limit. Furthemore, the report title GNN For Speech Recognition doesn't match the problem statement to classify music genre. While I am convinced by the justification of GNN, I'm not sure how the team will use Siamese Network to extract image feature. My understanding of the Siamese Network is that it output the similarity between input features. I assume it could also be use for classification which might make it redundant with using GNN.
